Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -117,21 +190,23 @@ def test_get_all_myschedule_objects_of_user(self):
             pytest.fail('Expected Json, got {0}.'.format(response.text))
         assert response.json()[0]["EVENT_ID"] == '1100'
 
-# Verify that an activity leader can create a Guest membership for someone.
+# Create a myschedule object of the currently logged in user.
 
 # Expectations:
 # Endpoints -- api/myschedule/
 # Expected Status Code -- 201 Created.
 # Expected Content -- A Json object with a GORDON_ID attribute.
 
+    
+
     def test_myschedule_post(self):
         session = None
         self.session = self.createAuthorizedSession(username, password)
         self.url = hostURL + 'api/myschedule/'
         self.data = {
-            'GORDON_ID' : '999999097',
-            'LOCATION' : 'KOSC 118',
-            'DESCRIPTION' : "Intro to Prog. Lab",
+            'GORDON_ID' : str(my_id_number),
+            'LOCATION' : location,
```

Comment:
If this is the only place the variable location and description is used, I would just hard code it into this test instead of making a global variable

Reply:
Just added global variables to the put test as well, thoguh those are probably the only two tests we need them for now.